cloud/guides/configuring-environments.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Configuring Environments
+description: Learn how to configure different aspects of your Flipt environments
+---
+
+## Commit Messages, Pull-Request Titles and Bodies
+
+Flipt Cloud dynamically generates commit messages and pull-requests based on parameters of the changes being made.
+It passes these parameters through Go templates to produce the resulting messages.
+While cloud is configured some sensible defaults, these can be overridden on a per Flipt environment basis.
+
+To override these templates you must provide a `flipt.yaml` file at the root of your environment tree.
+
+```console
+.
+└─ production
+    └─ flipt.yaml
+```
+
+<Info>
+  To recap: an environment is a combination of a Git `repository`, a `branch`
+  name and `directory` path. These are selected when you create your environment
+  in Flipt Cloud. The `flipt.yaml` must exist in this directory and branch in
+  order to configure the relevant environment.
+</Info>
+
+When this file isn't provided, a default one is implicitly generated. The default is effectively the following:
+
+```yaml
+version: "1.0"
+templates:
+  commit_message: |-
+    {{- if eq (len .Changes) 1 }}
+    {{- (index .Changes 0) }}
+    {{- else -}}
+    updated multiple resources
+    {{ range $change := .Changes }}
+    {{ $change }}
+    {{- end }}
+    {{- end }}
+  proposal_title: "Flipt: Update features in {{ .Base.Name }}"
+  proposal_body: |-
+    Update Flipt resources in [{{ .Base.Name }}]({{ .Base.HostURL }})
+    The branched environment can be viewed at [{{ .Branch.Name }}]({{ .Branch.HostURL }})
+```
+
+### Commit Message
+
+Commit message templates are supplied with a list of changes.
+Currently, Flipt only provides a single change.
+However, this may change in the future. As such, the template is provided a list of changes.
```

Comment:
```suggestion
Currently, Flipt only provides a single change, however, this may change in the future. As such, the template provides a list of changes.
```